xblock/plugin.py

```diff
@@ -173,7 +178,7 @@ def test_the_thing():
         def _decorator(func):                           # pylint: disable=C0111
             @functools.wraps(func)
             def _inner(*args, **kwargs):                # pylint: disable=C0111
-                global PLUGIN_CACHE
+                global PLUGIN_CACHE  # pylint: disable=global-statement
```

Comment:
Micro-nit: I prefer to tab the pragmas out to the right so they are less distracting.